src/controls/Switch.qml

```diff
@@ -23,6 +23,15 @@ import Material 0.3
 Controls.Switch {
     id: control
 
+    property Action action
```

Comment:
I prefer to use the checkable property; Imagine I set checkable to false and displayCheckableActionAsSwitch to true, the switch behavior will be weird